ui/src/store/device-connection.ts

```diff
@@ -36,11 +36,13 @@ export class DeviceConnection {
       const startRemoteConnectResult = await this.deviceControlStore.startRemoteConnect()
 
       startRemoteConnectResult.donePromise.then(({ data }) => {
-        const debugCommand = device.manufacturer === 'Apple' ? `curl http://${data}/status` : `adb connect ${data}`
+        this.debugCommand = device.manufacturer === 'Apple'
+          ? `curl http://${data}/status`
+          : device.manufacturer && device.model
+            ? `adb connect ${data}`
+            : `sdb connect ${data}`
```

Comment:
maybe add condition for tizen? We need to implement webos, vidaa and other things shortly